IBAnimatable/GradientDesignable.swift

```diff
@@ -78,14 +46,48 @@ fileprivate extension GradientDesignable where Self: UIView {
   }
 
   func configurePredefinedGradient() -> GradientColor? {
-
     guard let gradientType = predefinedGradient else {
       return nil
     }
     return gradientType.colors
   }
 }
 
+public extension GradientDesignable where Self: UINavigationBar {
+  public func configureGradient() {
+    let predefinedGradient = configurePredefinedGradient()
+    if let startColor = startColor, let endColor = endColor {
+      configureGradient(startColor: startColor, endColor: endColor)
+    } else if let startColor = predefinedGradient?.start, let endColor = predefinedGradient?.end {
+      configureGradient(startColor: startColor, endColor: endColor)
+    }
+  }
+}
+
+fileprivate extension GradientDesignable where Self: UINavigationBar {
```

Comment:
Why not keeping only one extension for `UINavigationBar` and making the methods `private`? I think it would be more readable to group all the related methods by UIKit components